IonDotnet/Tree/IIonValue.cs

```diff
@@ -1,9 +1,10 @@
 ﻿using System.Collections.Generic;
-using IonDotnet.Tree.Impl;
 
 namespace IonDotnet.Tree
 {
-    public interface IIonValue
+    public interface IIonValue : IIonInt, IIonFloat, IIonTimestamp, IIonDecimal,
```

Comment:
I think this list should be ordered in the same way that the binaryId is.

Reply:
Done.